lib/src/sync/window.dart

```diff
@@ -28,11 +28,11 @@ abstract class Windows {
 /// Upon use, the window will automatically be set as active.
 abstract class Window {
   /// The size of the window.
-  @Deprecated('JSON wire legacy support, emulated for newer browsers')
+  @Deprecated('JSON wire legacy support, unsupported for newer browsers')
```

Comment:
The JSON wire spec specifies this in terms of size and location, but W3C specifies it in terms of rect. So location/size are what people use now, and rect is what they should use in the future.

To use the same interface for all WebDriver tests (i.e. irrespective of browser) we need the API there at least, even if it's not hooked up.  I could emulate size and location for W3C, but as these are poorly specified anyway I want people to move to rect. (Also emulation would be a 50-100ms slower due to extra calls.) 

In practice, I suspect this is non issue -- I don't think anyone uses this much. But in the event that people do, they have a migration path and we can drop this later.